Automattic/wp-calypso
# Jetpack Plans: remove `Unlimited Storage Space` feature and link generic `All` features to jetpack.com.

Fixes https://github.com/Automattic/wp-calypso/issues/22597.

Addresses part of https://github.com/Automattic/wp-calypso/pull/23059.

**To test:**
- checkout this branch in your local Calypso,
- go to `http://calypso.localhost:3000/plans/:connected_site`, 
- verify the absence of `Unlimited Storage Space` feature in the Personal plan,
- verify that bottom `All * features` link to `https://jetpack.com/features/comparison/`

![screen shot 2018-03-07 at 21 39 33](https://user-images.githubusercontent.com/13561163/37119568-b60c154a-224f-11e8-9a4f-742f5531b7e9.png)
